packages/react/src/component-tests/IcButton/IcButton.cy.tsx

```diff
@@ -686,7 +686,7 @@ describe("IcButton visual regression tests in high contrast mode", () => {
   it("should render icon variants in high contrast mode", () => {
     mount(<IconButtonGroup />);
 
```

Comment:
Do we need to check that the component is hydrated before running the screenshot?

Reply:
Of course, my bad. Added those in now.